From 9ab2b96c2ee679c6dcb387174be211db5d70fa97 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Sun, 1 May 2016 15:35:32 +0200
Subject: [PATCH] Avoid intentional syntax error

This file has an intentional syntax error, meant to validate QA, but
it breaks byte compilation of this package.

Issue reported upstream:
https://github.com/crossbario/crossbar/issues/750.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 crossbar/worker/test/examples/syntaxerror.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/crossbar/worker/test/examples/syntaxerror.py b/crossbar/worker/test/examples/syntaxerror.py
index a5947cc..585c995 100644
--- a/crossbar/worker/test/examples/syntaxerror.py
+++ b/crossbar/worker/test/examples/syntaxerror.py
@@ -29,5 +29,3 @@
 #####################################################################################
 
 from __future__ import absolute_import, division
-
-class # noqa
-- 
2.7.4

